backend/tests/conftest.py

```diff
@@ -62,3 +67,73 @@ def override_get_db():
     app.dependency_overrides[get_db] = override_get_db
     yield TestClient(app)
     del app.dependency_overrides[get_db]
+
+
+@pytest.fixture()
+def create_user_instance(session):
+    new_user = UserObject(
+        username="Mike", email="mike@gmail.com", hashed_password="fb34jkfsdf", id=1
+    )
+    UserQuery().create_user(session, new_user)
+
+
+@pytest.fixture()
+def create_user_search_settings_instance(session):
+    new_settings = UserSearchSettingsObject(
+        country="Poland",
+        category="Sport News",
+        source="cnn",
+        language="French",
+        user_id=1,
+        id=1,
+    )
+    UserSearchSettingsQuery().create_user_search_settings(session, new_settings)
+
+
+@pytest.fixture()
+def create_multiple_user_instances(session):
+    new_user_one = UserObject(
+        username="Mike", email="mike@gmail.com", hashed_password="fb34jkfsdf", id=1
+    )
+    new_user_two = UserObject(
+        username="Bob", email="bob@gmail.com", hashed_password="sdsda4142", id=2
+    )
+    new_user_three = UserObject(
+        username="John", email="john@gmail.com", hashed_password="dgij314saf", id=3
+    )
+
+    UserQuery().create_user(session, new_user_one)
+    UserQuery().create_user(session, new_user_two)
+    UserQuery().create_user(session, new_user_three)
+
+
+@pytest.fixture()
+def create_multiple_users_search_settings_instance(session):
```

Comment:
I think we shouldn't use fixtures like those. It has fixed number of created instances. In this case 3. In my opinion fixtures should be reusable in more tests. It looks like it's tailored to your specific test when you assert if the number of rows in table is 3. 

`assert len(users_search_settings) == 3`

Normal factory function like 
`def create_users_search_settings: -> UserSearchSettingsObject
`
should be enough